.github/workflows/schedule_grype.yaml

```diff
@@ -0,0 +1,12 @@
+name: Grype security scan
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 10 * * * '
+
+jobs:
+  grype-scan:
+    name: Scan project for vulnerabilities
+    uses: ./.github/workflows/grype_scan.yaml
```

Comment:
What's the value of splitting out the grype part of this from the scheduler portion?

Reply:
In this case, not that much now. For this example, right now, I *could* move it back into the schedule.

Initially, it was running both on schedule and as part of the build, where it was useful to have this to avoid duplication.

More generally, I was also thinking about whether it's worth having potentially common workflows stored elsewhere, so that there's one central place to maintain them. A reusable workflow or composite action would be one way of doing this.